### ISSUE TITLE
JIT fuser result of dropout doesn't fully match eager mode 

### ISSUE THREAD

**Issue by ailzhang** [pytorch/pytorch#40882]
Running this test with legacy executor should repro:
```
    def test_dropout_cuda(self):
        # Dropout AD is dispatched to _fused_dropout in CUDA case,
        # which is not included in TestJitGeneratedFunctional
        
        x = torch.ones((4, 4)).cuda().requires_grad_()

        with enable_profiling_mode_for_profiling_tests():
            @torch.jit.script
            def func(x):
                return torch.nn.functional.dropout(x)


            with freeze_rng_state():
                out = func(x)
                #grad = torch.autograd.grad(out.sum(), x)

            with freeze_rng_state():
                out_ref = torch.nn.functional.dropout(x)
                #grad_ref = torch.autograd.grad(out_ref.sum(), x)

            print("out=", out)
            print("out_ref=",out_ref)
            self.assertEqual(out, out_ref)
            self.assertEqual(grad, grad_ref)
```
cc: @ngimel @Krovatkin

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
